editor/src/app/sites/settings/site-settings.component.ts

```diff
@@ -23,7 +23,7 @@ import { SettingModel, SettingChildModel, SettingConfigModel, SettingGroupConfig
     <div class="setting-group"
          [class.is-expanded]="camelifySlug(currentGroup) === settingGroup.slug"
          *ngFor="let settingGroup of settings$ | async">
-      <h3 [routerLink]="['/settings', slugifyCamel(settingGroup.slug)]" queryParamsHandling="preserve" class="hoverable">
+      <h3 [routerLink]="['/settings', camelifySlug(currentGroup) === settingGroup.slug ? '' : slugifyCamel(settingGroup.slug)]" queryParamsHandling="preserve" class="hoverable">
```

Comment:
Let's improve usability of this one as well - make this in to a link:
- add `role="link"`
- Angular adds the `tabindex` automatically
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_link_role